### PR TITLE
[Feature] Add multi machine `dist_train`.

### PR DESCRIPTION
**Pull request by linfangjian01**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add more detailed documentation for users of single and multi-machine training to suit different development environments.

ref: https://github.com/open-mmlab/mmselfsup/pull/232

## Modification

Add the English and Chinese description documents of startup tasks. Modify the tools/dist_train.sh and tools/dist_test.sh files.

### CODE REVIEW

**Review by MeowZheng — COMMENTED** [review]
please modify the tools/dist_train.py

**Comment by MengzhangLI** [comment]
Those files should be modified:

- [ ] docs/en/get_started.md
- [ ] docs/zh_cn/get_started.md
- [ ] tools/dist_train.sh
- [ ] tools/dist_test.sh

**Review by MeowZheng — COMMENTED** [review]
also modify docs/cn/train.md

File: docs/en/train.md [hunk 1]
```diff
@@ -17,12 +17,14 @@ Equivalently, you may also use 8 GPUs and 1 imgs/gpu since all models using cros
 
 To trade speed with GPU memory, you may pass in `--cfg-options model.backbone.with_cp=True` to enable checkpoint in backbone.
 
-### Train with a single GPU
+### Train on single machine
```
Comment (MeowZheng) [inline docs/en/train.md:20]: Train on single a machine

File: docs/en/train.md [hunk 2]
```diff
@@ -59,47 +61,109 @@ Optional arguments are:
...
+
+On the second machine:
+
+```shell
+NNODES=2 NODE_RANK=1 PORT=$MASTER_PORT MASTER_ADDR=$MASTER_ADDR sh tools/dist_train.sh $CONFIG $GPUS
+```
+
+Usually it is slow if you do not have high speed networking like InfiniBand.
+
+#### Manage jobs with Slurm
```
Comment (MeowZheng) [inline docs/en/train.md:114]:
```suggestion
### Manage jobs with Slurm
```

File: docs/zh_cn/train.md [hunk 3]
```diff
@@ -49,48 +51,109 @@ export CUDA_VISIBLE_DEVICES=-1
...
+NNODES=2 NODE_RANK=1 PORT=$MASTER_PORT MASTER_ADDR=$MASTER_ADDR sh tools/dist_train.sh $CONFIG $GPUS
 ```
 
-您可以查看 [slurm_train.sh](../tools/slurm_train.sh) 以熟悉全部的参数与环境变量。
+但是，如果您不使用高速网路连接这几台机器的话，训练将会非常慢。
 
-如果您多个机器已经有以太网连接， 您可以参考 PyTorch
-[launch utility](https://pytorch.org/docs/stable/distributed.html#launch-utility) 。
-若您没有像 InfiniBand 这样高速的网络连接，多机器训练通常会比较慢。
+#### 使用slurm管理任务
```
Comment (MeowZheng) [inline docs/zh_cn/train.md:104]:
```suggestion
### 使用slurm管理任务
```

**Review by MeowZheng — APPROVED** [review]

**Pull request merged**